packages/appstore/src/components/empty-state/empty-state.tsx

```diff
@@ -7,7 +7,7 @@ type TProps = {
     renderTitle: () => React.ReactNode;
 };
 
-const EmptyState: React.FC<TProps> = ({ icon_name, renderMessage, renderTitle }) => {
+const EmptyState = ({ icon_name, renderMessage, renderTitle }: TProps) => {
```

Comment:
@thisyahlen-deriv Not related to this PR, But I think this component can be replaced with the one in `@deriv/cashier` 🤔 So maybe we can move that one to `@deriv/components` and reuse it in both places 🤔

Reply:
alrightss